.doc_gen/metadata/iot_metadata.yaml

```diff
@@ -0,0 +1,33 @@
+# zexi 0.4.0
+iot_DescribeEndpoint:
+  title: Return a unique endpoint specific to the &AWS; account making the call using an &AWS; SDK
+  title_abbrev: Get endpoint
+  synopsis: get an &IoT; endpoint.
```

Comment:
Move extra info out of title into synopsis.

Reply:
I've trimmed the text.